### ISSUE TITLE
Modified Horton Infiltration Bug

### ISSUE THREAD

**Issue by LRossman** [USEPA/Stormwater-Management-Model#158]
Line 545 in the modHorton_getInfil() function of infil.c should be changed from
```
        if ( infil->Fmax > 0.0 ) infil->Fe = MAX(infil->Fe, infil->Fmax);
```
to
```
        if ( infil->Fmax > 0.0 ) infil->Fe = MIN(infil->Fe, infil->Fmax);
```

**Comment by cbuahin** [comment]
Recall that the modified Horton infiltration method was updated to limit the maximum infiltration in issue #59 to reflect changes that failed to make it into the main branch as part of PR #56 for the [v5.1.15 update ](https://github.com/USEPA/Stormwater-Management-Model/pull/56/commits/f2df1d2648f6fc4acb56360b04683059e07bb05b ).

While the rationale for the fix suggested in this issue is clear enough to infer, it will be useful to provide some clarification as to why these changes are being suggested for our understanding as folks tasked with maintaining and advancing SWMM and for community understanding. 

If the previous fix addresses the issue raised here, I will go ahead and close this issue.

**Comment by LRossman** [comment]
No, this is a different bug not addressed in PR #56. As seen on page 103 of the Hydrology Reference Manual,

![image](https://github.com/USEPA/Stormwater-Management-Model/assets/12769926/ca985c4a-e09f-444c-89cd-3601d6a3f172)

the updated cumulative infiltration should be the **smaller** of the newly computed value and the max. limit, not the larger of the two.

**Comment by cbuahin** [comment]
Understood. Thanks Lew.
